frank/default_parameters.json

```
@@ -42,5 +46,10 @@
     "force_style"      : true,
     "bin_widths"       : [1e3, 5e4],
     "iter_plot_range"  : null
+  },
+
+  "analysis" : {
+    "bootstrap"        : false,
+    "n_trials"         : null
```

Again, I don't like there being two not obviously linked parameters. 

```suggestion
  "bootstrap" : {
    "on"               : false,
    "n_trials"         : null
```